lib/MusicBrainz/Server/Entity/Medium.pm

```diff
@@ -142,6 +126,21 @@ has 'cdtoc_track_count' => (
     isa => 'Int',
 );
 
+has 'cdtoc_track_lengths' => (
+    is => 'rw',
+    isa => 'ArrayRef[Maybe[Int]]'
```

Comment:
Are these types correct? I always suck at moose typing.

Reply:
I think they're okay; the values can be undef if the track durations aren't set, right?